Sources/MapboxMaps/Viewport/Transitions/Default/DefaultViewportTransitionOptions.swift

```diff
@@ -0,0 +1,7 @@
+public struct DefaultViewportTransitionOptions: Hashable {
+    public var maxDuration: TimeInterval
```

Comment:
this is maxDurationMs on Android, we could see if there's a need for alignment before rc.